### DISCUSSION TITLE
Locked out of my own organization

### DISCUSSION THREAD

**Discussion by Kaprosamurai** [community/community#40126] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hello, I'm trying to use this site's services but have run into a problem.

I've made an organization with my same username 'Kaprosamurai' but I'm apparently not a part of my organization as I'm not able to see anything on it, and following the instructions given by the Github's help pages I'm apparently not a part of any organization, but considering I made the organization I should have been automatically made a part of it. I'm unable to make another organization as the name is already taken since I've already made it.

I discovered this when I was trying to add another person to the organization. The organization was made (or at least attempted to be made) by either me or my brother while we were making our accounts and yet neither of us are counted as even being part of any organization, and yet the organization name is already taken apparently so it had to have been created successfully. Yet Github has no way of even searching for it, please help.

**Comment by JCDiana** [comment]
Did get your token yet? Or have 2Fa

**Comment by airtower-luna** [comment]
You can't make an organization with the same name as an existing user. So the likely explanation is simply that creating the organization failed, and that's why you're not part of any organization. Try creating an organization with a name that's not taken already.

**Comment by jfullstackdev** [comment]
so you can't log in anymore ? it seems you turned your personal account into an Org

it's a different thing when you simply create an Org vs. turning your personal account into an Org

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization

**Comment by ajdichmann** [comment]
I am having the same problem, I turned a personal account into an organization - but now cant access the organization or log into the original personal account
